README.md

```diff
@@ -1,4 +1,13 @@
-JMEDAS
+JMEValidator
 ======
 
-JME POG CMS Data Analysis School (CMSDAS) exercise
+Common framework for CMS JEC / JER analyzes.
+
+### Dependencies
+
+This package depends on [TreeWrapper](https://github.com/blinkseb/TreeWrapper). Inside the `src` folder of you CMSSW release, do:
+
+```sh
+git clone git@github.com:blinkseb/TreeWrapper.git JMEAnalysis/TreeWrapper
+scram b -j8
+```
```

Comment:
Can you add the ability to set the branch status or get a specific entry to your TreeWrapper class? This will certainly become useful if/when these trees become large. Also, at least one of my JEC codes already requires the SetBranchStatus method. I know I'm looking to the future here.